docs/source/usage/reference.rst

```diff
@@ -50,3 +50,475 @@ Thank you and enjoy PIConGPU and our community!
+If you want to add your publication to the list as well please feel free to contact us or open a pull request directly.
+
+Application of PIConGPU in various physics scenarios
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the following, a list of publications describing using PIConGPU is given in various cases.
+
+Laser plasma electron acceleration
+""""""""""""""""""""""""""""""""""
+ - Laser wakefield acceleration (LWFA) with self-truncated ionization-injection (STII) [Couperus2017]_
+ - PhD thesis on experimental aspects of LWFA with STII [Couperus2018]_,
+ - PhD thesis on theoretical ascpects of self-focusing during LWFA with STII [Pausch2019]_
+ - Hybrid laser-driven/beam-driven plasms acceleration [Kurz2021]_
+
+Laser plasma ion acceleration
+"""""""""""""""""""""""""""""
+ - Proton acceleration from cryogenic hydrogen jets [Obst2017]_
+ - Mass-Limited, Near-Critical, micron-scale, spherical targets [Hilz2018]_
+ - PhD thesis on theoretical aspects of mass-limited, near-critical, micron-scale, spherical targets [Huebl2019]_
+ - All-optical stuctering of laser-accelerated protons [Obst-Huebl2018]_
+ - PhD thesis on laser-driven proton beam structering [Obst-Huebl2019]_
+ - Laser-ion multi-species acceleration [Huebl2020]_
+
+Laser plasma light sources and diagnostics
+""""""""""""""""""""""""""""""""""""""""""
+ - PhD thesis on radiation from LWFA [Pausch2019]_
+ - Laser-driven x-ray and proton sources [Ostermayr2020]_
+ - Betatron x-ray diagnostic for beam decoherence [Koehler2019]_, [Koehler2021]_
+
+
+Astrophysics
+""""""""""""
+ - Simulating the Kelvin Helmholtz instability (KHI) [Bussmann2013]_
+ - Visualizing the KHI [Huebl2014]_
+ - Theoretical model of the radiation evolution during the KHI [Pausch2017]_
+ - PhD thesis covering the KHI radiation [Pausch2019]_
+
+Machine Learning
+""""""""""""""""
+
+
+Methods used in PIConGPU software
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the following, a list of references is given, sorted by topics, that describe PIConGPU as a software.
+References to publications of implemented methods that were developed by other groups are also given for completeness.
+These references are marked by an asterisk.
+
+
+General code references
+"""""""""""""""""""""""
+
+ - First publication on PIConGPU [Burau2010]_
+ - Currently main reference [Bussmann2013]_
+ - Most up-to-date reference [Huebl2019]_
+
+Field solvers
+"""""""""""""
+ - Yee field solver\ :sup:`\*` [Yee1966]_
+ - Lehe fields solver\ :sup:`\*` [Lehe2013]_
+ - High-Order Finite Difference solver\ :sup:`\*` [Ghrist2000]_
+
+Particle pushers
+""""""""""""""""
+ - Boris pusher\ :sup:`\*` [Boris1970]_
+ - Vay pusher\ :sup:`\*` [Vay2008]_
+ - Reduces Landau-Lifshitz pusher\ :sup:`\*` [Vranic2016]_
+
+Current deposition
+""""""""""""""""""
+ - Esirkepov method\ :sup:`\*` [Esirkepov2001]_
+ - Villasenor and Buneman method\ :sup:`\*` [Villasenior1991]_
```

Comment:
@PrometheusPi is the idea of this section to reference all e.g. pushers and field solvers we implement, or only "most important" ones?

Reply:
@sbastrakov It would be ideal to reference all pushers/algorithms that we use.

Reply:
Forgot `HigueraCary` particle pusher

Reply:
@sbastrakov I added the Higuera-Cary pusher.